src/Native/SymSgdNative/CMakeLists.txt

```diff
@@ -4,16 +4,11 @@ set(SOURCES
     SymSgdNative.cpp
 )
 
-if(WIN32)
-    find_library(MKL_LIBRARY MklImports HINTS ${CMAKE_SOURCE_DIR}/../../packages/mlnetmkldeps/0.0.0.5/runtimes/win-x64/native)
-else()
+find_library(MKL_LIBRARY MklImports HINTS ${MKL_LIB_PATH})
+if(NOT WIN32)
     list(APPEND SOURCES ${VERSION_FILE_PATH})
-    if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
-        message("Linking SymSgdNative with MKL on macOS.")
-        find_library(MKL_LIBRARY libMklImports.dylib HINTS "${CMAKE_SOURCE_DIR}/../../packages/mlnetmkldeps/0.0.0.5/runtimes/osx-x64/native")
-    else()
-        message("Linking SymSgdNative with MKL on linux.")
-        find_library(MKL_LIBRARY libMklImports.so HINTS ${CMAKE_SOURCE_DIR}/../../packages/mlnetmkldeps/0.0.0.5/runtimes/linux-x64/native)
+    if(NOT APPLE)
+        message("executing commands for linux.")
```

Comment:
We should remove this before merging.

Reply:
yikes. I had these for debugging logs but I will remove it. Thanks!